src/app/_services/auth.service.ts

```diff
@@ -5,14 +5,13 @@ import { Http, Headers, RequestOptions, Response } from '@angular/http';
 export class AuthService {
 
   private url = 'http://localhost:3000/api/login';
-  private TOKEN: string = 'token';
```

Comment:
can you explain why you removed the 'string' part?

Reply:
Because token: string = "token" is redundant, so Tslint warns about this by giving a no-inferrable-type.